### DISCUSSION TITLE
過去のすべての記事が今のOrganizationへ紐付けられて困っています

### DISCUSSION THREAD

**Discussion by bigwheel** [increments/qiita-discussions#316] [要望・アイデア・不具合]
過去の記事すべてが今の組織へ紐付けられて困っています。
おそらく[Qiita Organizationをアップデートします \- Qiita Blog](https://blog.qiita.com/organization-update/)
の変更でそういった処理をしたのだと思いますが、私は3年前に[今のOrganization](https://qiita.com/tags/speee)へ参加しており、逆に言うとそれ以前の記事は別のOrganizationか私個人に紐づくべきものです。後者の個人のものが今のOrganizationへ紐づくのはともかく、以前所属したOrnigazation(端的に言えば前職や前々職)で書いた記事が現在の会社に紐づくものとして扱われるのは大いに問題に感じます。

一つ一つ手動で紐づけを解除していっても良いですが、[83記事](https://qiita.com/bigwheel)もあるので手間です。
一律すべて解除でもいいので、記事とOrganizationの紐づけを一括して解除する方法があれば教えてください。

**Comment by kato647** [comment]
@bigwheel 
ご不便をおかけして申し訳ございません
以下の項目をサポートフォームに送っていただければ対応させていただきます。

### サポートフォーム

https://support.qiita.com/hc/ja/requests/new

### 件名

記事のOrganizationの紐付け解除の件

### 説明

1. ユーザー名
2. 紐付けを外したい「組織・団体名」
3. 紐付けを外したい「Organization URL」
4. 紐付けを外したい記事

```
例
1. @qiita_id
2. Qiita株式会社
3. https://qiita.com/organizations/qiita-inc
4. 投稿日が2019年1月1日以前の記事の紐付けを解除してください（2019年1月1日の記事も含む）
```

**Reply by bigwheel** [reply]
了解しました。サポートフォームより依頼を投げさせていただきます。

**Reply by tanaka-asami415** [reply]
@bigwheel 
フォームへご連絡いただきありがとうございました。こちら対応が完了いたしました！🙇

**Reply by bigwheel** [reply]
迅速に対応していただきありがとうございます、こちらで2019/10/31以前の記事の紐付けが解除されていることを確認しました。
<img width="1191" alt="スクリーンショット 2022-10-13 14 05 12" src="https://user-images.githubusercontent.com/826646/195506376-7795b9b9-be8d-471a-b2d7-7c0505f6df02.png">
